components/camel-milo/src/main/java/org/apache/camel/component/milo/client/MonitorFilterConfiguration.java

```diff
@@ -37,8 +37,8 @@ public class MonitorFilterConfiguration implements Cloneable {
     @UriParam(defaultValue = "0.0", description = "Deadband value for MonitorFilterType DataChangeFilter.")
     private Double dataChangeFilterDeadbandValue = 0.0;
 
-    @UriParam(description = "Daata change trigger for data change monitor filter type.")
-    private DataChangeTrigger dataChangeFilterTrigger = DataChangeTrigger.StatusValue;
+    @UriParam(description = "Data change trigger for data change monitor filter type.")

```

Comment:
```suggestion
    @UriParam(defaultValue = "StatusValueTimestamp", description = "Data change trigger for data change monitor filter type.")

```
Let's add the default value to the metadata too